swan-lake/development-tutorials/observability/overview-of-ballerina-observability.md

```diff
@@ -0,0 +1,204 @@
+
+### Setting up Ballerina runtime configurations
+To enable observability (both metrics and tracing) in the Ballerina runtime, use the following configurations:
+```toml
+[ballerina.observe]
+enabled = true
+provider = <PROVIDER>
+```
+
+Metrics and tracing can be enabled separately as well by using the following configurations. Add additional configurations specific to the tool or platform you are using.
+```toml
+[ballerina.observe]
+metricsEnabled=true
+metricsReporter=<METRICS_REPORTER>
+tracingEnabled=true
+tracingProvider=<TRACING_PROVIDER>
+```
+
+Configuration key | Description | Default value | Possible values 
+--- | --- | --- | --- 
+ballerina.observe. metricsEnabled | Whether metrics monitoring is enabled (true) or disabled (false) | false | `true` or `false`
+ballerina.observe. metricsReporter | Reporter name that reports the collected Metrics to the remote metrics server. This is only required to be modified if a custom reporter is implemented and needs to be used. | `None` | `prometheus`, `newrelic` or if any custom implementation, the name of the reporter.
+ballerina.observe.tracingEnabled | Whether tracing is enabled (true) or disabled (false) | false | `true` or `false`
+ballerina.observe.tracingProvider | The tracer name, which implements the tracer interface. | `None` | `jaeger`, `zipkin`, `newrelic` or the name of the tracer of any custom implementation.
+
+## Observability tools and platforms supported by Ballerina
+
+This outlines how to enable and configure observability in Ballerina for various tools and platforms. It provides a step-by-step guide for setting up monitoring, tracing, and logging using widely used observability solutions.
+
+Observability tools and platforms help monitor and analyze application performance, identify issues, and ensure reliability. Following are the main observability tools and platforms supported by Ballerina:
+
+- **[Prometheus](https://prometheus.io/):** A monitoring system and time-series database for metrics collection and alerting.
+
+- **[Jaeger](https://www.jaegertracing.io/):** A distributed tracing platform for monitoring and debugging microservices.
+
+- **[Zipkin](https://zipkin.io/):** A distributed tracing system to collect and look up trace data.
+
+- **[New Relic](https://newrelic.com/):** A full-stack observability platform for application performance monitoring (APM) and telemetry.
+
+- **[DataDog](https://www.datadoghq.com/):** A cloud-based observability service offering monitoring, metrics, traces, and logging.
+
+- **[Elastic Stack](https://www.elastic.co/elastic-stack):** A collection of tools (Elasticsearch, Logstash, Kibana) for centralized logging and analytics.
+
+Following contains the guide to set up and observe Ballerina programs in each of the observability tool or platform mentioned above.
+
+- [Observe Ballerina programs with Prometheus](/learn/supported-observability-tools-and-platforms/prometheus)
```

Comment:
Can we reword the titles with what exactly we are montoring with each system? E.g. `Observe metrics using Prometheus`

Reply:
@anupama-pathirage There are some platforms (DataDog and New Relic) which provides both metrics and tracing. Shall we mention both in the title?

Reply:
+1 to mention both